app/components/transfer/navigation/TransferNavigation.scss

```diff
@@ -0,0 +1,11 @@
+.component {
+  display: flex;
+  height: 100%;
+  justify-content: space-between;
+}
+
+.navItem {
+  height: 100%;
+  width: percentage(1/2); // Update when adding new nav items!
```

Comment:
You can set `width: 100%` for the exact same result and even remove `justify-content` in the `component` if you want. Flex will make sure it works

Reply:
Yeah this is better. Also changed `WalletNavigation.scss`.